### ISSUE TITLE
measuring speed

### ISSUE THREAD

**Issue by SzotsGabor** [jonathanmcdermid/Clovis#2]
I don't know how to do it. I tried go depth n but whatever number n is it seems to disregard it. No nps, no time output. There is hit% which I think is hash hit rate but what is mpn?

**Comment by jonathanmcdermid** [comment]
hit% was hashtable hitrate and mps was a metric I was using for seeing the average number of moves done at each node. I've removed them now. Release 1.2 has the depth command functionality and time.

**Comment by SzotsGabor** [comment]
Hi Jonathan,

It seems it still does not support 'go depth n', in fact does not support 'go infinite' either because the search is stopped at depth 11 or 12 (depending on which compile I use).

**Comment by jonathanmcdermid** [comment]
In 1.2 the default time is not infinite, so you would need to enter something like 'go depth n wtime 5000 btime 5000'.

**Comment by SzotsGabor** [comment]
To my knowledge, 'go depth n' should make the engine calculate until it reaches depth n (search n plies only). 'Go infinite' should make the engine calculate until the "stop" command.

Anyway, I tried your suggestion and after go depth 15 wtime 100000 btime 100000 it indeed searched up to 15 plies. However, all other engines I know will calculate until ply 15 even if I don't include the wtime btime parameters.

I tried Shredder, which we might consider as authentic. Go depth 30 made it search till ply 30, go depth 30 wtime 100 btime 100 made it stop after 8 plies.
